src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -14,6 +14,7 @@
     <DebugType>portable</DebugType>
     <DebugSymbols>true</DebugSymbols>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <EnableXlfLocalization>false</EnableXlfLocalization>
```

Comment:
@adamsitnik can this be set? It appears to solve my local issues with having to manually generate the XLF files and then ignore during commit.

Do we have localized resources?

Reply:
@steveharter as long as it makes your dev experience better and the CI is green, it's perfectly fine

> Do we have localized resources?

afaik not. All resources were added to avoid having big strings defined as `const string`?